components/organisms/BiddingTable.tsx

```diff
@@ -23,12 +26,13 @@ const columns = [
   'Highest Bid',
   'My Bid',
   'Status',
-  '', // reserve last column for any button actions
+  'Input', // reserve last column for any button actions
 ]
 
 export default function BiddingTable({ data }: Props): JSX.Element {
   console.log('data: ', data) // use data variable so eslint doesnt complain
   return (
+    <a href="./dashboard">
```

Comment:
1. why are you wrapping the table? 
2. is husky working on your computer? because it's not fixing formatting / recommit issue (indentation)